best-practices/development-tools/azure/devops/deployment/aws-elasticbeanstalk/aws-elastic-beanstalk-deployment.md

```diff
@@ -0,0 +1,77 @@
+# Deploying into AWS Elastic Beanstalk
+
+## Pre-Deployment
+### Install AzureDevOps Extensions
+There are two extensions should be installed before in AzureDevOps [AWS Toolkit for Azure DevOps](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.aws-vsts-tools&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList) and [ReplaceTokens](https://marketplace.visualstudio.com/items?itemName=citrus-andriessen.cag-replace-tokens&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList).
```

Comment:
How do you feel about making these extentions mandatory intead of optional? It seems like the user will need these two to move forward on the below instructions. 

```suggestion
Please install the following two extensions:
* AzureDevOps [AWS Toolkit for Azure DevOps](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.aws-vsts-tools&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList) 
* [ReplaceTokens](https://marketplace.visualstudio.com/items?itemName=citrus-andriessen.cag-replace-tokens&targetId=af056fb1-e0c2-4678-9ac3-dee687452af2&utm_source=vstsproduct&utm_medium=ExtHubManageList).
```

Reply:
Nice suggestion